tests/testthat/test-drive_ls.R

```diff
@@ -28,8 +36,17 @@ test_that("drive_ls() outputs contents of folder", {
   skip_on_appveyor()
   skip_on_travis()
 
-  expect_equivalent(
-    drive_ls(nm_("foo")),
-    dribble()
-  )
+  ## path
+  out <- drive_ls(nm_("foo"))
+  expect_s3_class(out, "dribble")
+  expect_identical(out$name, c(nm_("a"), nm_("b")))
+
+  ## dribble
+  d <- drive_path(nm_("foo"))
+  out2 <- drive_ls(d)
+  expect_identical(out[c("name", "id")], out2[c("name", "id")])
+
+  ## id
+  out3 <- drive_ls(as_id(d$id))
+  expect_identical(out[c("name", "id")], out3[c("name", "id")])
```

Comment:
I think we've been under-attending to re: path vs dribble vs id. I realized that once I created `is_path()`. We've been / are at risk of file ids being interpreted as file paths.